fuel/transformers/__init__.py

```diff
@@ -60,6 +63,24 @@ def get_epoch_iterator(self, **kwargs):
         self.child_epoch_iterator = self.data_stream.get_epoch_iterator()
         return super(Transformer, self).get_epoch_iterator(**kwargs)
 
+    def get_data(self, request=None):
+        if self.batch_input:
+            return self.get_data_from_batch(request)
+        else:
+            return self.get_data_from_example(request)
+
+    def get_data_from_example(self, request=None):
+        raise NotImplementedError(
+            "`{}` does not support examples as inputs, '\
```

Comment:
Python actually automatically concatenates strings within parentheses, so you can do:
```python
raise NotImplementedError(
    "`{}` does not support examples as inputs, "
    "but `batch_input` was set to `False`".format(type(self))
)
```